core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -150,17 +150,17 @@ public void seekTo(VideoPosition position) throws IllegalStateException {
     @Override
     public void stop() {
         reset();
+        listenersHolder.getStateChangedListeners().onVideoStopped();
     }
 
     @Override
     public void release() {
-        reset();
+        stop();
```

Comment:
We just had a conversation about it 😆 we should be ok, we have tests backing everything in this class and if we ever wanted to change the implementation of `stop()` we can also change `release()` at the same time. Not that we think we will anyways.